app/main.py

```diff
@@ -1 +1,54 @@
-# write your code here
+class SoftwareEngineer:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    skills = []
+
+    def learn_skill(self, skill: str) -> None:
+        self.skills.append(skill)
+
+
+class FrontendDeveloper(SoftwareEngineer):
+    def __init__(self, name: str) -> None:
+        super().__init__(name)
+
+    skills = ["JavaScript", "HTML", "CSS"]
```

Comment:
```suggestion
class FrontendDeveloper(SoftwareEngineer):
    def __init__(self, name: str) -> None:
        super().__init__(name)
        self.skills += ["JavaScript", "HTML", "CSS"]
```
And the same for others

Reply:
Sorry, but I can't do that, because in this case I didn't passed the tests. Please, check README

Reply:
You can do this if you fix skills attribute according to previous comment

Reply:
It isn't fixed